plugins/gatsby-transformer-home-example-code/gatsby-node.js

```diff
@@ -3,7 +3,6 @@
  */
 
 const crypto = require('crypto');
-const path = require('path');
```

Comment:
An error was detected in another PR

https://github.com/reactjs/ja.reactjs.org/pull/456/files
![image](https://user-images.githubusercontent.com/43776161/145740492-26921f34-d359-4840-8830-dd670194d666.png)